openedx/features/content_tagging/fixtures/system_defined.yaml

```diff
@@ -1,22 +1,40 @@
+- model: content_tagging.taxonomyorg
+  pk: -1
+  fields:
+    taxonomy: -1
+    org: null
+    rel_type: OWN
 - model: oel_tagging.taxonomy
   pk: -2
   fields:
-    name: Organizations
-    description: Allows tags for any organization ID created on the instance.
+    name: Content Authors
+    description: Allows tags for any user ID created on the instance.
     enabled: true
     required: true
     allow_multiple: false
     allow_free_text: false
     visible_to_authors: false
     _taxonomy_class: openedx.features.content_tagging.models.ContentAuthorTaxonomy
+- model: content_tagging.taxonomyorg
+  pk: -2
+  fields:
+    taxonomy: -2
+    org: null
+    rel_type: OWN
 - model: oel_tagging.taxonomy
   pk: -3
   fields:
-    name: Content Authors
-    description: Allows tags for any user ID created on the instance.
+    name: Organizations
+    description: Allows tags for any organization ID created on the instance.
     enabled: true
     required: true
     allow_multiple: false
     allow_free_text: false
     visible_to_authors: false
     _taxonomy_class: openedx.features.content_tagging.models.ContentOrganizationTaxonomy
+- model: content_tagging.taxonomyorg
+  pk: -3
+  fields:
+    taxonomy: -3
+    org: null
+    rel_type: OWN
```

Comment:
Instances which have already applied the [0003_system_defined_fixture migration](https://github.com/openedx/edx-platform/pull/32907/files#diff-0172eb4157513c0cc393426d1bac3251578472e8aae2f5fe6dcb0ef8b7f31762) (like edx.org) won't receive these fixture changes.

Instead of updating the existing migration, we need to add a new migration which applies these changes.

@ChrisChV  can you work with @rpenido  to come up with a strategy for applying changes to these fixtures over time?

Reply:
If this file is just creating 4 rows of data, make it a data migration, not a fixture at all.

Reply:
Hi @ChrisChV!
I removed the changes made in 0003_*. I created a new migration 0004_ to add the TaxonomyOrg.
ToDo: Check if we will remove the Org and Author taxonomies in this PR or elsewhere.

Reply:
@rpenido Thanks! Looks good! @bradenmacdonald is right, we can create the taxonomies on the `0003_` migration without the fixture. Could you update that on this PR if you have time?

> ToDo: Check if we will remove the Org and Author taxonomies in this PR or elsewhere.

Yep, I think will be on a future PR. FYI, the [Author taxonomy is necessary by product](https://www.figma.com/file/DmUC2e7BAoQsGDq4KgApPa?node-id=1519:86157&mode=design#514994738)

Reply:
Done 56300b7dbc593566ae4dbcc10d8058572c360f54!